frontend.bat

```diff
@@ -459,6 +459,7 @@ set vswfilter=!vswPriority!
 
 :_vswAttempt
 call :dbgprint "attempts with filter: " vswfilter vswPreRel
+call :dbgprint " using: " vswbin
```

Comment:
I've checked both VS 15.9.28307.586 & VS preview 16.1.28822.285 but I can't reproduce your mentioned error with non-evaluated value. 

Moreover, in fact, we have both same instructions inside single scope (I mean without an additional `call` processing, setlocal etc). Only `goto` below, see [this region](https://github.com/gerhardol/hMSBuild/blob/ac8cc908670b77130247e15322137cf80c76fa16/frontend.bat#L452-L462)

To be sure, Is it still reproducible for you after removing [this](https://github.com/gerhardol/hMSBuild/blob/ac8cc908670b77130247e15322137cf80c76fa16/frontend.bat#L462) ?
can you also attach log with `-debug` key

Reply:
I removed it now and it works for me
It was working briefly before I saw the problem too. Cannot see what triggers it.